### ISSUE TITLE
twisted.conch.test.test_insults fails on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8698]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8698|
|Type|enhancement|
|Created|2016-07-27 23:30:07Z|
|Branch|https://github.com/twisted/twisted/tree/8698-rodrigc-conch-py3|

If I do:

```
python3 bin/trial twisted.conch.test.test_insults
```

it fails on Python 3.

<details><summary>Searchable metadata</summary>

```
trac-id__8698 8698
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8698_rodrigc_conch_py3 8698-rodrigc-conch-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1469662207199948 1469662207199948
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/434

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_3"></a>|@acabhishek942 commented|
|-|-|

looks good to me

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset ef1b5509e10d52e2019cd577b44d2101d8d974d1
```
#!CommitTicketReference repository="" revision="ef1b5509e10d52e2019cd577b44d2101d8d974d1"
Merge pull request #434 from twisted/8698-rodrigc-conch-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8698

Port twisted.conch.test.test_insults to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
